code/src/components/Directions.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import { getAction } from 'reducers/labyrinth'
+import styled from 'styled-components/macro'
+
+const Directions = () => {
+  const actions = useSelector((store) => store.labyrinth.actions)
+  const [isActive, setActive] = useState(false)
+
+  const dispatch = useDispatch()
+
+  const toggleDisplay = () => {
+    setActive(!isActive);
+  }
+
+  return (
+    <Direction>
+      <DirectionBtn1 className={isActive ? 'hidden-btn' : null} type="button" onClick={toggleDisplay}>Directions</DirectionBtn1>
```

Comment:
Kind of the reverse of my last comment :), I remember reading that you could use className={isActive && 'hidden-btn'} instead of using null!